tutorial_dataset_2.qmd

```diff
@@ -407,8 +407,10 @@ custom_R_code: '
     mutate(
       across(c("TRAIT Leaf Dry Mass UNITS g"), ~na_if(.x,0))
     ) %>%
-    group_by(name_original) %>% #column with taxon name
-      across(c("TRAIT Growth Form CATEGORICAL EP epiphyte (mistletoe) F fern G grass H herb S shrub T tree V vine"), replace_duplicates_with_NA) %>%
+    group_by(name_original) %>% 
```

Comment:
I had the note so people realised that `name_original` was the input column name, not an output column - did you find it not useful or confusing how I'd annotated it?